### DISCUSSION TITLE
title bar and window

### DISCUSSION THREAD

**Discussion by gnuchanos** [qtile/qtile#3968] [Q&A]
how ı open title bar 
![2022-11-08-124107_314x34_scrot](https://user-images.githubusercontent.com/117280480/200530236-aea0f1e4-709d-4d04-9b08-07388e4ac40e.png)

and some programs is floating but some not why ?
![2022-11-08-124202_1280x1012_scrot](https://user-images.githubusercontent.com/117280480/200530406-f41043d8-cce9-429b-b879-fff1b859f242.png)
![2022-11-08-124209_1280x1012_scrot](https://user-images.githubusercontent.com/117280480/200530415-ae08c55f-bd01-424c-b458-e12d38fd9b34.png)

**Comment by m-col** [comment]
> how ı open title bar

Title bars are the responsibility of the program that opens the window. Qtile doesn't make title bars.

> and some programs is floating but some not why ?

Most likely they are matching the `float_rules` that you are passing to your `Floating` layout. If you get information about your window (you can use `xprop` to get information about a window) you can see if it matches any of your float rules.

**Reply by gnuchanos** [reply]
xprop?  what
did you mean gnome,i3,openbox not make title bar ? how it works then?

**Reply by m-col** [reply]
Those WMs draw title bars. Qtile does not have this feature. It is not a simple feature, but if somebody wanted to add it I think it would be welcome.
